source/fundamentals/serialization/poco.txt

```diff
@@ -599,6 +599,8 @@ serialization-related attributes:
 - ``[BsonRepresentation()]``, which specifies serialization of the ``Price`` field as a BSON ``Double`` type
 - ``[BsonDefaultValue()]``, which sets the ``Name`` property to
   ``"Generic item"`` if no value has been assigned to it
+- ``[BsonDateTimeOptions(DateOnly = true)]``, which specifies that our DateTime property
+  only represents a date value with no associated time
```

Comment:
[Style Guide on where to place "only"](https://www.mongodb.com/docs/meta/style-guide/terminology/alphabetical-terms/#o:~:text=Place%20only%20preceding)

```suggestion
- ``[BsonDateTimeOptions(DateOnly = true)]``, which specifies that the ``DateTime`` property
  represents only a date value, with no associated time
```